JitHub.AuthFunction/GithubAuth.cs

```diff
@@ -73,6 +73,6 @@ public static string ProcessRequest(HttpRequest req)
             return new BadRequestObjectResult("Bad Request, Github request error");
         }
 
-        return new OkObjectResult(new { token = token });
+        return new OkObjectResult(token);
```

Comment:
**could do this in the next iteration** but we only need to return the token string, not the entire object. Same for the client side and the protocol trigger.

Reply:
changed to just token string